src/test/scala/com/cloudera/sparkts/TimeSeriesRDDSuite.scala

```diff
@@ -227,4 +229,69 @@ class TimeSeriesRDDSuite extends FunSuite with LocalSparkContext with ShouldMatc
       new DenseVector(Array(9.0, 11.0))
     ))
   }
+  test("lagWithIncludeOriginals")
+  {
+    val conf = new SparkConf().setMaster("local").setAppName(getClass.getName)
+    TimeSeriesKryoRegistrator.registerKryoClasses(conf)
+    sc = new SparkContext(conf)
+    val vecs = Array(0 until 10, 10 until 20, 20 until 30)
+      .map(_.map(x => x.toDouble).toArray)
+      .map(new DenseVector(_))
+      .map(x => (x(0).toString, x))
+    val start = new DateTime("2015-4-9",DateTimeZone.UTC)
+    val index = uniform(start, 10, 1.days)
+    val rdd = new TimeSeriesRDD(index, sc.parallelize(vecs))
+    val lagTSrdd = rdd.lag(2,true)
```

Comment:
To be consistent with how variables are capitalized elsewhere in the code, `lagTSrdd` -> `lagTsrdd`